src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/JsonContent.netcoreapp.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http.Json
+{
+    public partial class JsonContent
+    {
+        protected override Task SerializeToStreamAsync(Stream stream, TransportContext context, CancellationToken cancellationToken)
+            => JsonSerializer.SerializeAsync(stream, Value, ObjectType, _jsonSerializerOptions, cancellationToken);
```

Comment:
I can't find it now, but I saw in some comment you mentioned a concern about some validation SerializeAsync is doing that's happening now later than it was before.  If that's a concern, such validation could be manually added to the JsonContent constructor, assuming the validation is simple and cheap.

Reply:
The concerns were: 
* Type shouldn't be null 
* Type.IsAssignableFrom(value.GetType()) 

Both will occur until we send the content; should we duplicate the validation so it occurs when we create the `JsonContent`? @terrajobst